conda_build/metadata.py

```diff
@@ -289,7 +289,7 @@ def parseNameNotFound(error) -> str:
 @cache
 def evalidate_model():
     model = base_eval_model.clone()
-    model.nodes.extend(["Call", "Attribute"])
+    model.nodes.extend(["Call", "Attribute", "Tuple", "List", "Dict"])
```

Comment:
This was an oversight on my end, sorry about the breakage. I thought including them in allowed_functions was enough 😬